### PR TITLE
fix(chat): fix card view data update (Issue #2224)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Information on card view is now updated (name, description, version, icon)

Issues:

- Issue #2224 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2249.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — DISMISSED** [review]

File: apps/chat/src/components/Marketplace/TabRenderer.tsx [hunk 1]
```diff
@@ -194,6 +196,15 @@ export const TabRenderer = ({ isMobile }: TabRendererProps) => {
     [setDetailsModel],
   );
 
+  useEffect(() => {
+    if (!detailsModel) return;
+    setDetailsModel(
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Marketplace/TabRenderer.tsx]: Lets update `detailsModel` inside `handleEdit` method

Comment (Derikyan) [reply apps/chat/src/components/Marketplace/TabRenderer.tsx]: inside handleEdit is the entity before editing

**Comment by Derikyan** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/522434)
>Environment URL: https://chat-ai-dial-chat-pr-2249.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2249.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**